eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/SiteAccessAware/ConfigurationProcessor.php

```diff
@@ -52,8 +57,6 @@ public function __construct(ContainerInterface $containerBuilder, $namespace, $s
      *
      * Important: Available SiteAccesses need to be set before ConfigurationProcessor to be constructed by a bundle
      * to set its configuration up.
-     *
-     * @param array $availableSiteAccesses
```

Comment:
AFAICS PhpStorm does not consider this as very strict, so it reports that PHPDoc is missing.